core-war/src/main/webapp/util/javaScript/vuejs/components/comments/silverpeas-comments-templates.jsp

```diff
@@ -0,0 +1,208 @@
+        <silverpeas-comment v-for="comment in comments"
+                            v-bind:key="comment.id"
+                            v-bind:current-user="user"
+                            v-bind:comment="comment"
+                            v-bind:readonly="false"
+                            v-on:comment-update="updateComment"
+                            v-on:comment-deletion="deleteComment">
+        </silverpeas-comment>
+      </silverpeas-fade-transition-group>
+
+    </div>
+
+    <div v-if="user.anonymous && messages.anonymousViewMode === 'counter'">
+
+      <div class="commentsCount">
+        <span>{{ infoNbOfComments }}</span>
+      </div>
+
+      <div class="connection" v-if="user.anonymous">
+        <div class="buttons">
+          <button class="button logOn" onclick="top.location.href='${loginUrl}'">
+            {{ messages.loginAction }}
+          </button>
+        </div>
+      </div>
+
+    </div>
+
+  </div>
+</silverpeas-component-template>
+
+<!-- ###############################################################################################
+Edition block of a new comment.
+################################################################################################ -->
+<silverpeas-component-template name="comment-edition">
+  <div id="edition-box">
+    <p class="title"><fmt:message key="comment.add"/></p>
+    <div class="avatar">
+      <img v-bind:src="currentUser.avatar" alt="avatar">
+    </div>
+    <textarea id="comment-edition-text"
+              class="text"
+              v-bind:value="value"
+              v-on:input="$emit('input', $event.target.value)">
+    </textarea>
+    <div class="buttons">
+      <button class="button" v-on:click="$emit('comment-adding', comment)">
+        <fmt:message key="GML.validate"/>
+      </button>
+    </div>
+  </div>
+</silverpeas-component-template>
+
+<!-- ###############################################################################################
+A comment with its content, the date it has been authored and the name of the author.
+According to the rights of the current user, the comment can be updated and deleted.
+################################################################################################ -->
+<silverpeas-component-template name="comment">
+  <div v-bind:id="'comment' + comment.id" class="oneComment">
+    <div>
+      <div class="action">
+        <img v-if="canBeUpdated"
+             v-on:click="$emit('comment-update', comment)"
+             src="${updateIcon}" alt="${commentUpdateAction}">
+        <span>&nbsp;</span>
+        <img v-if="canBeDeleted"
+             v-on:click="$emit('comment-deletion', comment)"
+             src="${deletionIcon}" alt="${commentDeletionAction}">
+      </div>
+      <div class="avatar">
+        <img v-bind:src="comment.author.avatar" alt="avatar">
+      </div>
+      <p class="author">
+        <span class="name" v-bind:class="{ userToZoom : displayUserZoom}"
```

Comment:
Maybe a space is missing after `displayUserZoom` in terms of code formatting.

Reply:
Yes